### PR TITLE
[zh-cn]sync containers/images.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/concepts/containers/images.md

### CODE REVIEW

File: content/zh-cn/docs/concepts/containers/images.md [hunk 1]
```diff
@@ -846,6 +846,37 @@ If you need access to multiple registries, you can create one secret for each re
 -->
 如果你需要访问多个仓库，可以为每个仓库创建一个 Secret。
 
+<!--
+## Legacy built-in kubelet credential provider
+
+In older versions of Kubernetes, the kubelet had a direct integration with cloud provider credentials.
+This gave it the ability to dynamically fetch credentials for image registries.
+-->
+## 遗留的内置 kubelet 凭据提供程序
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/containers/images.md:855]:
```suggestion
## 旧版的内置 kubelet 凭据提供程序
```

File: content/zh-cn/docs/concepts/containers/images.md [hunk 2]
```diff
@@ -846,6 +846,37 @@ If you need access to multiple registries, you can create one secret for each re
...
+
+在旧版本的 Kubernetes 中，kubelet 与云提供商凭据直接集成。
+这使它能够动态获取镜像仓库的凭据。
+
+<!--
+There were three built-in implementations of the kubelet credential provider integration:
+ACR (Azure Container Registry), ECR (Elastic Container Registry), and GCR (Google Container Registry).
+-->
+kubelet 凭据提供程序集成的三个内置实现：
+ACR（Azure 容器仓库），ECR（Elastic 容器仓库）和 GCR（Google 容器仓库）
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/containers/images.md:865]:
```suggestion
kubelet 凭据提供程序集成存在三个内置实现：
ACR（Azure 容器仓库）、ECR（Elastic 容器仓库）和 GCR（Google 容器仓库）
```

**Review by windsonsea — COMMENTED** [review]
legacy 旧版，感觉读起来更通顺一些

**Review by ydFu — APPROVED** [review]
/lgtm

**Pull request merged**